docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -267,25 +268,9 @@ Make sure you're up to date and your working directory is completely clean:
 > **Note:** To throw away any and all changes to tracked and untracked files you
 > can run `git clean -dfx ; git reset --hard`.
 
-Create a new branch named `merge-develop-into-master` with both *master* and
-*develop* as ancestors and with the exact contents from *develop*:
+Create a pull request from `develop` into `master` and merge it.
```

Comment:
Add "No reviewer or approval is required for this update" as today.

Reply:
done